src/Main.elm

```diff
@@ -27,19 +30,21 @@ main =
 
 type Page
     = Home
-    | Auth
+    | Auth (Maybe String)
```

Comment:
the `Maybe String` represent the query parameter `jwt` on the auth endpoint.
This endpoint can be `/auth` then jwt is `Nothing` or `/auth?jwt=...` then the value is `Just ...`